instances/latest/terminologies/cellType/mainOlfactoryBulbGranuleNeuron.jsonld

```diff
@@ -0,0 +1,22 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/cellType/mainOlfactoryBulbGranuleNeuron",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+  "definition": "'Main olfactory bulb granule neurons' are axonless inhibitory interneurons and form the majority of neurons in the vertebrate main olfactory bulb [adapted from [Egger et al. (2003)](https://doi.org/10.1523%2FJNEUROSCI.23-20-07551.2003)].",
```

Comment:
new convention? "Axonless inhibitory interneurons that form the majority of neurons in the vertebrate main olfactory bulb ..."

Reply:
OR "A subclass of neurons, axonless and inhibitory, situated in the  the main olfactory bulb of vertebrates."